source/query-bar.txt

```diff
@@ -69,7 +69,7 @@ as in the following filter example:
    { "_id": { "$oid": "58eee4f43d1a10a4e5339aac" } }
 
 When you have entered a valid MongoDB query, the :guilabel:`Filter`
-label is no longer red and :guilabel:`Apply` button is enabled. 
+label is no longer red and :guilabel:`Find` button is enabled. 
 
 Project
 ~~~~~~~
```

Comment:
Note how you wrote [Line 85](https://github.com/mongodb/docs-compass/pull/12/files#diff-93adb5021a0a047de98a19b6c0073135R85) vs. [Lines 94-95]https://github.com/mongodb/docs-compass/pull/12/files#diff-93adb5021a0a047de98a19b6c0073135R94-95). I think that the latter is better, i.e. just the colon and no "for example".